host/xtest/regression_4000.c

```diff
@@ -1009,6 +1009,186 @@ static const uint8_t hash_data_shake256_out1[] = {
 	0x9f, 0x87, 0x26, 0xe4, 0x62, 0xa1
 };
 
+/*
+ * More SHA-3 test cases from
+ * https://csrc.nist.gov/projects/cryptographic-algorithm-validation-program/secure-hashing
+ * https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Algorithm-Validation-Program/documents/sha3/sha-3bytetestvectors.zip
+ */
+
+/* Excerpt from SHA3_224LongMsg.rsp */
+static const uint8_t hash_data_sha3_224_in2[] = {
+	0x31, 0xc8, 0x2d, 0x71, 0x78, 0x5b, 0x7c, 0xa6, 0xb6, 0x51, 0xcb, 0x6c, 0x8c, 0x9a, 0xd5, 0xe2,
```

Comment:
These lines are too long, please keep them under 80, with an exception for the links of course.